.github/workflows/matplotlib-tests.yml

```diff
@@ -0,0 +1,75 @@
+# Run tests and integration tests on Mac, which are triggered by each master push.
+# Currently, Python3.8 is only used as an environment.
+# This is mainly for the sake of speed.
+name: matplotlib-tests
+
+on:
+  pull_request:
+    paths:
+      - tests/visualization_tests/matplotlib_tests/**
+      - .github/workflows/matplotlib-tests.yml
+      - optuna/visualization/**.py
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  matplotlib-tests:
+    runs-on: ubuntu-latest
+
+    # Scheduled Tests are disabled for forks.
+    if: (github.event_name == 'schedule' && github.repository == 'optuna/optuna') || (github.event_name != 'schedule')
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Setup Python3.8
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.8
+
+    - name: Setup cache
+      uses: actions/cache@v4
+      env:
+        cache-name: test
+      with:
+        path: ~/Library/Caches/pip
+        key: ${{ runner.os }}-3.8-${{ env.cache-name }}-${{ hashFiles('**/pyproject.toml') }}-v1
+        restore-keys: |
+          ${{ runner.os }}-3.8-${{ env.cache-name }}-${{ hashFiles('**/pyproject.toml') }}
```

Comment:
In my understanding, this cache is defined for [mac-test](https://github.com/optuna/optuna/blob/master/.github/workflows/mac-tests.yml). Could you replace this with the ubuntu version, e.g., https://github.com/optuna/optuna/blob/master/.github/workflows/tests-with-minimum-versions.yml#L43-L51.